spec/actors/hyrax/actors/file_set_actor_spec.rb

```diff
@@ -136,10 +136,14 @@
   describe '#create_content when from_url is true' do
     before do
       expect(JobIoWrapper).to receive(:create_with_varied_file_handling!).with(any_args).once.and_call_original
+      allow(VisibilityCopyJob).to receive(:perform_later).with(file_set.parent).and_return(true)
+      allow(InheritPermissionsJob).to receive(:perform_later).with(file_set.parent).and_return(true)
```

Comment:
I think we should plan to migrate job queuing tests to the `have_equeued` matcher. It looks like we're using stubbing across the board, so :+1: to this for now.